README.md

```diff
@@ -42,15 +42,12 @@
 To install the latest version:
 ```
 curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash && export SUPPRESS_GCLOUD_CREDS_WARNING=true
-./terra
+sudo mv terra /usr/local/bin
+# Talk to Verily production environment (as opposed to a test environment):
+terra server set --name=verily --quiet
 ```
 
-To install a specific version:
-```
-export TERRA_CLI_VERSION=0.106.0
-curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash
-./terra
-```
+To install a specific version: `export TERRA_CLI_VERSION=0.106.0`
```

Comment:
Don't you still need to run the curl command after setting the TERRA_CLI_VERSION variable?

Reply:
Changed to `To install a specific version, run before above commands:`